packages/web-components/src/components/va-on-this-page/va-on-this-page.tsx

```diff
@@ -5,6 +5,12 @@ import { consoleDevError } from '../../utils/utils';
  * This component will render links based on the content around it. It scans the document for any `<h2>`
  * elements that contain an `id` inside of an `<article>` and will create a list of links to the headings.
  */
+
+/**
+ * @componentName On this page
```

Comment:
Do things still work if this block is moved to the existing doc block above?

Reply:
I kind of like the idea of keeping it separate, just for clarity. Do we gain anything by combining the blocks?

Reply:
There's no real gain, it's just a subjective thing. I'm more used to any JSdoc or TSdoc tags being in the same doc block as the string description, like this:

https://github.com/department-of-veterans-affairs/component-library/blob/9b58a8c0216c9215dd8b5b31fb3d90dea9f29227/packages/web-components/src/components/va-file-input/va-file-input.tsx#L98-L102